src/main.ts

```diff
@@ -0,0 +1,48 @@
+import * as core from "@actions/core";
+import * as github from "@actions/github";
+import { Octokit } from "@octokit/rest";
+import { Value } from "@sinclair/typebox/build/cjs/value";
+import { envSchema, startStopSchema, PluginInputs } from "./types";
+import { startStopBounty } from "./plugin";
+
+export async function run() {
+  const payload = github.context.payload.inputs;
+
+  const env = Value.Decode(envSchema, process.env);
+  const settings = Value.Decode(startStopSchema, JSON.parse(payload.settings));
+
+  const inputs: PluginInputs = {
+    stateId: payload.stateId,
+    eventName: payload.eventName,
+    eventPayload: JSON.parse(payload.eventPayload),
+    settings,
+    authToken: payload.authToken,
+    ref: payload.ref,
+  };
+
+  await startStopBounty(inputs, env);
+
+  return returnDataToKernel(inputs.authToken, inputs.stateId, {});
```

Comment:
```suggestion
  return toKernel(inputs.authToken, inputs.stateId, {});
```

Perhaps this is more succinct and still clear.